services/src/modules/authentication/strategies/jwt.ts

```diff
@@ -18,10 +19,15 @@ export default async function (request: fastify.FastifyRequest): Promise<void> {
 
   // Verify audience
   if (issuerConfig.audience) {
-    const audience = Array.isArray(decodedJWT.payload.aud)
+    const jwtAudiences = Array.isArray(decodedJWT.payload.aud)
       ? decodedJWT.payload.aud.map(a => String(a))
       : Array.of(String(decodedJWT.payload.aud));
-    if (!audience.includes(issuerConfig.audience)) {
+    const configAudiences = Array.isArray(issuerConfig.audience)
```

Comment:
Why no something simpler like `[].concat(issuerConfig.audience)`?

Reply:
tried now.. `Type 'string' is not assignable to type 'ConcatArray<never>'`

Reply:
This works but it's ugly `([] as string[]).concat(issuerConfig.audience)`